how-to/common/public/windows/version/about.js

```diff
@@ -0,0 +1,50 @@
+function addVersion(versionType, currentVersion) {
+	const lstVersions = document.querySelector('#lstVersions');
+	const colHeaders = Array.from(lstVersions.children[0].children).map((header) => header.textContent);
+	const versionTypeLabel = document.createElement('span');
+	versionTypeLabel.textContent = versionType;
+
+	const versionLabel = document.createElement('span');
+	versionLabel.textContent = currentVersion;
+
+	const cellVersionType = document.createElement('div');
+	cellVersionType.dataset.name = colHeaders[0];
+	cellVersionType.append(versionTypeLabel);
+
+	const cellVersionLabel = document.createElement('div');
+	cellVersionLabel.dataset.name = colHeaders[1];
+	cellVersionLabel.append(versionLabel);
```

Comment:
Fallback to `unknown` if not available i.e. `versionLabel ?? "unknown"`

Reply:
done